Disable arc4random_buf and arc4random_uniform. These functions would
need libbsd as dependency for the arc4random function, but they are
already included in newer versions of that library.

--- clens-0.7.0-orig/include/clens/clens.h
+++ clens-0.7.0/include/clens/clens.h
@@ -73,8 +73,6 @@
 #include <stdlib.h>
 #define NO_UTIL_H
 
-#define NEED_ARC4RANDOM_BUF
-#define NEED_ARC4RANDOM_UNIFORM
 #define NEED_STRNVIS
 #define NEED_STRNUNVIS
 #define NEED_STRLCAT
